ledger-core/virtual/integration/method_test.go

```diff
@@ -1704,3 +1707,192 @@ func TestVirtual_CallContractTwoTimes(t *testing.T) {
+
+	executeDone := server.Journal.WaitStopOf(&execute.SMExecute{}, 2)
+
+	runnerMock := logicless.NewServiceMock(ctx, mc, func(execution execution.Context) string {
+		return execution.Request.CallSiteMethod
+	})
+	server.ReplaceRunner(runnerMock)
+
+	authService := authentication.NewServiceMock(mc)
+
+	authService.GetCallDelegationTokenMock.Set(func(outgoing reference.Global, to reference.Global, pn pulse.Number, object reference.Global) (c1 payload.CallDelegationToken) {
+		service := authentication.NewService(ctx, server.JetCoordinatorMock)
+		return service.GetCallDelegationToken(
+			outgoing, to, pn, object)
+	})
+	authService.HasToSendTokenMock.Return(true) // always send tokens
+	// skip token checks, in tests we have only one node, so we need to allow self signed tokens
+	authService.IsMessageFromVirtualLegitimateMock.Set(func(_ context.Context, payloadObj interface{}, sender reference.Global, pr pulse.Range) (mustReject bool, err error) {
+		return false, nil
+	})
+
+	server.ReplaceAuthenticationService(authService)
+
+	server.Init(ctx)
+	server.IncrementPulseAndWaitIdle(ctx)
+
+	typedChecker := server.PublisherMock.SetTypedChecker(ctx, mc, server)
+
+	var (
+		barIsolation = contract.MethodIsolation{
+			Interference: contract.CallTolerable,
+			State:        contract.CallDirty,
+		}
+		fooIsolation = contract.MethodIsolation{
+			Interference: contract.CallTolerable,
+			State:        contract.CallDirty,
+		}
+
+		classA        = gen.UniqueGlobalRef()
+		objectAGlobal = reference.NewSelf(server.RandomLocalWithPulse())
+
+		objectBGlobal = reference.NewSelf(server.RandomLocalWithPulse())
+
+		callOutgoing = server.RandomLocalWithPulse()
+
+		outgoingCallRef = reference.NewRecordOf(objectBGlobal, callOutgoing)
+	)
+
+	Method_PrepareObject(ctx, server, payload.Ready, objectAGlobal)
+	Method_PrepareObject(ctx, server, payload.Ready, objectBGlobal)
+
+	// add ExecutionMocks to runnerMock
+	{
+		outgoingCall := execution.NewRPCBuilder(outgoingCallRef, objectBGlobal).
+			CallMethod(objectAGlobal, classA, "Bar", []byte("123")).
+			SetInterference(barIsolation.Interference).
+			SetIsolation(barIsolation.State)
+		objectBExecutionMock := runnerMock.AddExecutionMock("Foo")
+		objectBExecutionMock.AddStart(
+			func(ctx execution.Context) {
+				t.Log("ExecutionStart [B.Foo]")
+				assert.Equal(t, objectBGlobal, ctx.Object)
+				assert.Equal(t, callOutgoing, ctx.Request.CallOutgoing)
+			},
+			&execution.Update{
+				Type:     execution.OutgoingCall,
+				Error:    nil,
+				Outgoing: outgoingCall,
+			},
+		)
+		objectBExecutionMock.AddContinue(
+			func(result []byte) {
+				t.Log("ExecutionContinue [B.Foo]")
+				require.Equal(t, []byte("finish A.Bar"), result)
+			},
+			&execution.Update{
+				Type:   execution.Done,
+				Result: requestresult.New([]byte("finish B.Foo"), objectBGlobal),
+			},
+		)
+
+		runnerMock.AddExecutionMock("Bar").AddStart(
+			func(ctx execution.Context) {
+				t.Log("ExecutionStart [A.Bar]")
```

Comment:
remove t.log or change it  
https://insolar.atlassian.net/wiki/spaces/DEV/pages/1431568423/Core+v2+-+Logger+in+tests

Reply:
switched to initialized logger from test context